subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3122,9 +3127,6 @@ static inline void event_fex_prep(struct ll_conn *conn)
 		/* procedure request acked, move to waiting state */
 		conn->llcp_feature.ack--;
 
-		/* use initial feature bitmap */
-		conn->llcp_feature.features_conn = LL_FEAT;
```

Comment:
As far as I can tell, there are some code paths in the slave role that can reach this point without having `features_conn` initialized. Are you sure this line is not needed?

Reply:
Isn't the conn object is being setup in ll_adv_enable() ?

Reply:
You are right. Thanks for pointing that out.